pandas/io/excel/_calamitereader.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+from io import (
+    BufferedReader,
+    BytesIO,
+)
+from pathlib import PurePath
+from tempfile import NamedTemporaryFile
+from typing import Any
+
+from python_calamine import (
+    get_sheet_data,
+    get_sheet_names,
+)
```

Comment:
Don't import an optional dependency like this. You can take a look at how other excel backends are imported.

Reply:
Yes, good spot. I will modify.

Reply:
> `python-calamite` is not even half-year old. There might be pushback.
> 
> `python-calamite` doesn't have a wheel for arm macOS. And the instance on CircleCI doesn't have Rust installed. You either need to get the author of `python-calamite` to produce a wheel for arm macOS or install the Rust toolchain on CircleCI.

Have opened an issue in the library https://github.com/dimastbk/python-calamine/issues/3. As you mentioned another option is installing the Rust toolchain on CircleCI.